core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -140,6 +147,59 @@ export class AdminBackendApiService {
       });
     });
   }
+
+  private _postAdminActionAsync(action:string, payload:Object): Promise<void> {
+    return this.http.post<void>(
+      AdminPageConstants.ADMIN_HANDLER_URL, { action, ...payload }).toPromise();
+  }
+
+  async startNewJobAsync(jobType: string): Promise<void> {
+    let action = 'start_new_job';
+    let payload = {
+      job_type: jobType
+    };
+    this._postAdminActionAsync(action, payload);
```

Comment:
are we expecting to have return statement here and elsewhere? (Have you tested the changes manually?)

Reply:
https://drive.google.com/file/d/1foxkOmA0kM42ITmQtbFeW3RNK4VRPoyn/view?usp=sharing
PTAL @DubeySandeep